boards/nxp/frdm_mcxn947/board.c

```diff
@@ -250,7 +247,7 @@ static int frdm_mcxn947_init(void)
 
 #if DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(flexcan0))
 	CLOCK_SetClkDiv(kCLOCK_DivFlexcan0Clk, 1U);
-	CLOCK_AttachClk(kPLL1_CLK0_to_FLEXCAN0);
+	CLOCK_AttachClk(kFRO_HF_to_FLEXCAN0);
```

Comment:
Have we confirmed the Zephyr CAN tests and samples pass with this clock change to the FlexCAN?

Reply:
after @hakehuang test wit can.timging sample, run success.